docs/f5xc_k8s_site/http_lb.md

```diff
@@ -60,9 +61,9 @@ The following settings should be modified from their default:
 
 Prefixes:
 
-- Inventory: /api/inventory
-- Recommendations: /api/recommendations
-- API: /api
+- /api/inventory: Inventory origin
```

Comment:
I'm concerned that a user will copy this URL verbatim which is not valid.  
Two options: 

1. move back to original prefixes
2. create a markdown table